### PR TITLE
feat: mirror nwjs

### PR DESCRIPTION
**Pull request by fengmk2**
closes https://github.com/cnpm/mirrors/issues/107

### CODE REVIEW

**Comment by fengmk2** [comment]
Wait for cnpmjs.org sync done https://cnpmjs.org/mirrors/nwjs/v0.12.0-alpha1/

File: config/index.js [hunk 1]
```diff
@@ -199,6 +199,17 @@ var config = {
       // for mirrors
       alwayNewDirIndex: 0,
     },
+    nwjs: {
+      name: 'nwjs',
+      category: 'nwjs',
+      enable: true,
+      // interval: ms('5m'),
+      disturl: 'https://nwjs.s3-us-west-2.amazonaws.com',
```
Comment (fengmk2) [inline config/index.js]: @evshiron using https now.

File: sync/nw.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    }
+    var name = match[1];
+    if (name[0] === '.' || name[0] !== 'v') {
+      // <Prefix>.tmp/</Prefix>
+      // <Prefix>live-build/</Prefix>
+      // <Prefix>sample/</Prefix>
+      continue;
+    }
+    // only sync version >= 0.12.0
+    if (name.split('.')[0] === 'v0' && parseInt(name.split('.')[1]) < 12) {
```
Comment (dead-horse) [inline sync/nw.js]: 判断一下数组长度吧

File: sync/nw.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    var name = match[1];
+    if (name[0] === '.' || name[0] !== 'v') {
+      // <Prefix>.tmp/</Prefix>
+      // <Prefix>live-build/</Prefix>
+      // <Prefix>sample/</Prefix>
+      continue;
+    }
+    var splits = name.split('.');
+    // only sync version >= 0.12.0
+    if (splits.length === 3 && splits[0] === 'v0' && parseInt(splits[1]) < 12) {
```
Comment (fengmk2) [inline sync/nw.js]: @dead-horse done

**Comment by dead-horse** [comment]
测试修一下？

**Comment by fengmk2** [comment]
fixed

**Comment by dead-horse** [comment]
+1

**Pull request merged**